support-workers/src/main/scala/com/gu/support/workers/lambdas/FailureHandler.scala

```diff
@@ -42,6 +42,10 @@ class FailureHandler(emailService: EmailService) extends Handler[FailureHandlerS
         FailedEmailFields.contribution(email = state.user.primaryEmailAddress, IdentityUserId(state.user.id))
       case _: SupporterPlus =>
         FailedEmailFields.supporterPlus(email = state.user.primaryEmailAddress, IdentityUserId(state.user.id))
+
+      case _: TierThree =>
+        // TODO: Do we need a new one here?
+        FailedEmailFields.supporterPlus(email = state.user.primaryEmailAddress, IdentityUserId(state.user.id))
```

Comment:
Is this going to be closer to GW given that we might want to pass delivery details in the email. 

This is a consideration that we can look at while looking at the email.